setup.py

```diff
@@ -91,7 +91,6 @@ def get_package_data(package):
     packages=get_packages(package),
     package_data=get_package_data(package),
     cmdclass={'test': PyTest},
-    install_requires=install_requires,
```

Comment:
That was poor merging on my part between the changes made on the edx fork versus upstream. I'll make a PR to remove that `install_requires` - though it won't actually affect anything in this version.